.gitignore

```diff
@@ -12,3 +12,4 @@ bookclub-islr.html
 bookclub-islr.knit.md
 bookclub-islr_cache
 *.html
+Chap_6_Demo.R
```

Comment:
Why don't you want to share the demo? It's ok for the repository to have extra files like this if it makes sense.